src/All/README.md

```diff
@@ -44,6 +44,13 @@ allGood([ 'nice', '00', null ])
 ```haskell
 All.empty :: () -> All
 ```
+<details>
+  <summary>Breakdown</summary>
+  <dl>
+    <dt>result</dt><dd><code>All</code> wrapping <code>true</code></dd>
```

Comment:
This does have an input of Unit or () as that is to the left of the arrow.
so it could be

<dt>unit</dt><dd>nothing, all arguments passed will be ignored</dd>

Reply:
Is that necessary?